CHANGELOG.md

```diff
@@ -23,6 +24,8 @@ FEATURES:
       or system killing Terraform.
   * **Math operations** in interpolations. You can now do things like
       `${count.index+1}`. [GH-1068]
+  * **New AWS SDK:** Move to `aws-sdk-go` (hashicorp/aws-sdk-go), 
+      a fork of the offical `awslabs` repo.
```

Comment:
Might want to mention intent to get in sync with upstream so as not to cause any undue distress.